test/integration/statistics_announcement_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class StatisticsAnnouncementTest < ActionDispatch::IntegrationTest
+
+  test "official statistics are rendered correctly" do
+    setup_and_visit_content_item('official_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert_has_component_metadata_pair("Release date", "20 January 2016 9:30am (confirmed)")
+  end
+
+  test "national statistics are rendered correctly" do
+    setup_and_visit_content_item('national_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert page.has_css?('.national-statistics-logo img')
+    assert_has_component_metadata_pair("Release date", "January 2016 (provisional)")
```

Comment:
The later. The examples happen to be a mix, presumably intentionally so we have both cases covered without creating a combinatorial set of examples.

I think it might be clearer if the test name was just the example name, to indicate it's exercising that _example_.